src/app/pages/applications/chart-releases/chart-releases.component.html

```diff
@@ -26,10 +26,15 @@
         </div>
 
         <div class="chart-button-row" fxLayout="row" fxLayoutAlign="end end" (click)="$event.stopPropagation()">
-          <button mat-button (click)="portal(item)" *ngIf="item.status === ChartReleaseStatus.Active && item.portals && item.portals.web_portal && item.portals.web_portal[0]"
+          <div class="ports" >
+		  <span class="ports-text"> {{ appService.getPorts(item) | lowercase }}</span>
+          </div>
+		  <div *ngFor="let portal of item.portals | keyvalue">
```

Comment:
Indentation is off.
Also let's move ngIf from down below higher, you can wrap this div with <ng-container *ngIf="item.status === ChartReleaseStatus.Active">

Reply:
> Indentation is off.

Shoot, yes... I'm used to my tab-to-spaces pre-commit hook running, will fix!
And yes indeed, it shouldn't show if it isn't active, just like the buttons...